cmd/modelcmd/confirmation.go

```diff
@@ -4,40 +4,24 @@
 package modelcmd
 
 import (
-	"github.com/juju/cmd/v3"
 	"github.com/juju/gnuflag"
 
 	"github.com/juju/juju/environs/config"
 )
 
-// ConfirmationCommandBase provides common attributes and methods that
-// commands require to confirm the execution.
-// TODO (jack-w-shaw): When confirm-removal config item defaults to true, we can combine these
-
 // DestroyConfirmationCommandBase provides common attributes and methods that
 // commands require to confirm the execution of destroy-* commands
 type DestroyConfirmationCommandBase struct {
-	assumeYes      bool // DEPRECATED
 	assumeNoPrompt bool
 }
 
 func (c *DestroyConfirmationCommandBase) SetFlags(f *gnuflag.FlagSet) {
-	f.BoolVar(&c.assumeYes, "y", false, "Do not ask for confirmation. Option present for backwards compatibility with Juju 2.9")
-	f.BoolVar(&c.assumeYes, "yes", false, "")
 	f.BoolVar(&c.assumeNoPrompt, "no-prompt", false, "Do not ask for confirmation")
 }
 
-// Run implements Command.Run
-func (c *DestroyConfirmationCommandBase) Run(ctx *cmd.Context) error {
```

Comment:
If it's never Run, shouldn't we remove the command `DestroyConfirmationCommandBase` as well?

Reply:
great! I see now. Thanks